backend/dist/controllers/Material.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
```

Comment:
I am gonna do some google searching but i think we should gitignore the dist folder